src/api/comms.test.js

```diff
@@ -38,6 +38,12 @@ describe('checkStatus', () => {
     const json = jest.fn(() => data);
     expect(checkStatus({ ok: true, json })).toEqual(data);
   });
+  
+  it('returns headers on successful create', () => {
```

Comment:
unit test failing due to missing params to `checkStatus`: https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_dashboard/153/pull-tekton-dashboard-unit-tests/1129066520287121409/?log#log

PR open to ensure build fails as expected on node.js unit test failure: https://github.com/tektoncd/dashboard/pull/159

Unit test should look something like:

```js
it('returns headers on successful create', () => {
  const status = 201;
  const headers = { fake: 'headers' };
  expect(checkStatus({ ok: true, headers, status }).toEqual(headers);
});
```

Reply:
Thanks @AlanGreene I have done that update